src/types/package.rs

```diff
@@ -92,6 +92,7 @@ pub enum PackageType {
     Nuget,
     Cargo,
     Golang,
+    Sbom,
```

Comment:
I guess I might be a bit confused about how SBOM works, but isn't it just a format for listing dependencies that doesn't have its own registry?

These types are all their own ecosystem, some of which with multiple lockfiles (like Python). Is SBOM really its own ecosystem?

Reply:
It does not have its own registry. It's essentially an ecosystem that can contain multiple ecosystems.

I thought we were going to treat it as a first class citizen, (hence its own ecosystem?), and thus needing its own lockfile parser where the overall PackageType is `SBOM`. Scanning the lockfile can return multiple packages from various ecosystems, and we extract the ecosystems we support for analysis.

It is confusing and I've questioned it myself a few times on the best approach. So there will probably be more discussion for CLI PR.

Reply:
Hm I think that might just be something the CLI parser needs to be adjusted for. I would expect it to spit out packages for each individual ecosystem. We can't just erase the original ecosystem and replace it with `SBOM` because otherwise we can't submit them for analysis (since we won't know where they came from). So I don't think using `SBOM` as an ecosystem would work.

Reply:
I wouldn't be erasing the original ecosystem. It'd have the overall package type as an `SBOM`, each package would have the package type associated with it.

Reply:
> It'd have the overall package type as an `SBOM`

I think we need to get rid of the idea of an "overall package type" before we can make the SBOM parser work.

That's probably going to require some large changes to the `lockfile` crate (mainly because of the [`package_type`](https://github.com/phylum-dev/cli/blob/main/lockfile/src/lib.rs#L144-L145) function that is required for parsers. But that's a problem to fix there, not here.

Reply:
It shouldn't be to difficult to make these changes, considering that internally we already have all the package types stored correctly anyway. But I do agree with @kylewillmon that it would likely make sense to do this first.

Reply:
Removed the `SBOM` package type.